lib/pages/image_viewer/image_viewer.dart

```diff
@@ -25,8 +25,23 @@ class ImageViewer extends StatefulWidget {
 }
 
 class ImageViewerController extends State<ImageViewer> {
+  late TransformationController transformationController;
```

Comment:
why need `late` ?

Reply:
took a snipet, I removed it and still work

Reply:
I mean you should create it directly
```suggestion
  final _transformationController = TransformationController();
```